wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/rapidreactcommandbot/subsystems/Pneumatics.java

```diff
@@ -49,17 +49,13 @@ private double getPressure() {
    * @return command
    */
   public Command disableCompressorCommand() {
+    // Disable closed-loop mode on the compressor.
```

Comment:
This comment should be just about `m_compressor::disable`, not the whole `startEnd()` call.